app/services/observer-manager.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-scroll-modifiers/services/observer-manager';
```

Comment:
Never re-export services in addons. They should be injected with proper namespacing instead to avoid collisions.

Reply:
This one was interesting because the build was not recognizing the service until I re-exported in the `app`

Reply:
That's because you weren't injecting it with the addon namespace, as I noted below.